client/state/login/test/selectors.js

```diff
@@ -28,10 +28,14 @@ import {
 	getSocialAccountLinkService,
 } from '../selectors';
 
+const EMPTY_STATE = {
```

Comment:
Is it a reasonable expectation for these tests that state should always have a `login` key, and it is initialised to an object?